scheduler-driver/src/test/java/gov/nasa/jpl/aerie/scheduler/PrioritySolverTest.java

```diff
@@ -252,7 +252,7 @@ public void getNextSolution_coexistenceGoalOnActivityWorks_withInitialSimResults
     adHocFacade.insertActivitiesIntoSimulation(makePlanA012(problem).getActivities());
     adHocFacade.computeSimulationResultsUntil(problem.getPlanningHorizon().getEndAerie());
     final var simResults = adHocFacade.getLatestDriverSimulationResults().get();
-    problem.setInitialPlan(makePlanA012(problem), Optional.of(simResults));
+    problem.setInitialPlan(makePlanA012(problem), Optional.of(simResults), null);
```

Comment:
can't you get the id map from the `adHocFacade` and pass it ?

Reply:
done